packages/main/src/themes/sap_horizon_hcb/Switch-parameters.css

```diff
@@ -143,4 +143,30 @@
 
 	--_ui5_switch_icon_width: 1rem;
 	--_ui5_switch_icon_height: 1rem;
+}
+
+[data-ui5-compact-size],
+.ui5-content-density-compact,
+.sapUiSizeCompact {
+	--_ui5_switch_width: 3rem;
+	--_ui5_switch_min_width: none;
+	--_ui5_switch_with_label_width: 3.75rem;
+	--_ui5_switch_root_outline_top: 0.25rem;
+	--_ui5_switch_root_outline_bottom: 0.25rem;
+	--_ui5_switch_transform: translateX(100%) translateX(-1.375rem);
+	--_ui5_switch_transform_with_label: translateX(100%) translateX(-1.875rem);
+	--_ui5_switch_rtl_transform: translateX(1.375rem) translateX(-100%);
+	--_ui5_switch_rtl_transform_with_label: translateX(1.875rem) translateX(-100%);
```

Comment:
Instead of creating a new variable for rtl scenario only use a shared one.

For the mixed scenario where Cozy/Compact and RTL are just make on another variable that should be changed dynamically

```CSS
### Before
--_ui5_switch_rtl_transform_with_label: translateX(1.875rem) translateX(-100%);

### After
--_ui5_switch_rtl_transform_with_label: translateX(--my-new-variable) translateX(-100%);
```

Reply:
As we discussed it's not working as I expected so ignore my previous comment,